modules/ding_item_viewer/ding_item_viewer.install

```diff
@@ -0,0 +1,196 @@
+<?php
+/**
+ * @file
+ */
+
+/**
+ * Implements hook_install().
+ */
+function ding_item_viewer_install() {
+  module_enable(array('ding_more_info'));
+
+  // Create field group.
+  $group = (object) array(
+    'identifier' => 'group_data|ting_object|ting_object|ding_item_viewer_big',
```

Comment:
The identifier is build as group name, entity type, entity bundle, view mode. So, the first ting_object means entity_type and the second ting_object means entity_bundle.